classes/kernel.oeclass

```diff
@@ -248,18 +249,20 @@ UIMAGE_KERNEL_OUTPUT = ""
 UIMAGE_KERNEL_OUTPUT:USE_kernel_uimage = "arch/${KERNEL_ARCH}/boot/uImage"
 KERNEL_OUTPUT += "${UIMAGE_KERNEL_OUTPUT}"
 
-CLASS_FLAGS += "kernel_external_dtb kernel_dtc kernel_dtc_flags kernel_dts"
+CLASS_FLAGS += "kernel_external_dtb kernel_dtc kernel_dtc_flags kernel_dts_vendor kernel_dts"
 DEFAULT_USE_kernel_dtc_flags = "-R 8 -p 0x3000"
 
 KERNEL_DTS_DIR = "arch/${KERNEL_ARCH}/boot/dts/"
 KERNEL_DTS_DIR:USE_kernel_dts_dir = "${USE_kernel_dts_dir}"
 KERNEL_DTS_FILE = "${MACHINE}.dts"
 KERNEL_DTS_NAME = "${MACHINE}"
+KERNEL_DTB_VENDOR = ""
 KERNEL_DTB_FILE = ""
 
 addhook kernel_devicetree_init to post_recipe_parse after set_useflags
 def kernel_devicetree_init(d):
     kernel_dtb = d.get('USE_kernel_external_dtb')
+    kernel_dts_vendor = d.get('USE_kernel_dts_vendor')
```

Comment:
Without any setting of USE_kernel_dts_vendor, will we end up with None?  And how is that handled later on?